test/ng/directive/ngOptionsSpec.js

```diff
@@ -779,6 +779,23 @@ describe('ngOptions', function() {
     expect(options[2]).not.toBeMarkedAsSelected();
   });
 
+
+  it('should render the options only one time', function() {
+    scope.values = ['black', 'white', 'red'];
+    createSelect({
+      'ng-model': 'value'
+    });
+
+    element.attr('ng-options', 'value for value in values');
+    var repaint = 0;
+    element[0].addEventListener('DOMSubtreeModified', function(ev) {
+      repaint++;
+      expect(repaint).toBeLessThan(2);
```

Comment:
Firefox 47 failed this expect. Probably because `DOMSubtreeModified`behaves differently in it. Maybe we should use MutationObserver instead for Browsers that support it?